app/scripts/controller-init/messengers/notifications/notification-services-controller-messenger.ts

```diff
@@ -14,7 +14,6 @@ export function getNotificationServicesControllerMessenger(
     name: 'NotificationServicesController',
     allowedActions: [
       // Keyring Actions
-      'KeyringController:withKeyring',
```

Comment:
This is mistake this has to be re-enabled given that we removed some of the controllers bumps from the initial PR.

I'll revert this.